apps/gamify/api/src/app/app.controller.ts

```diff
@@ -1,13 +1,29 @@
+import { DataService } from '@gamify/data';
 import { Controller, Get } from '@nestjs/common';
 
 import { AppService } from './app.service';
 
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
+  constructor(
+    private readonly appService: AppService,
+    private readonly dataService: DataService
+  ) {}
 
   @Get()
   getData() {
     return this.appService.getData();
   }
+  
+  @Get('test')
+  async testAction() {
+
+    // this.dataService.createUser({ email: "selderman@move4mobile.com" });
```

Comment:
Only keep this, if you don't mind you work email address ending up in the codebase.